crud/select/comparators.lua

```diff
@@ -127,15 +158,18 @@ function comparators.gen_func(cmp_operator, key_parts)
     return func
 end
 
-function comparators.gen_tuples_comparator(cmp_operator, key_parts)
-    local keys_comparator, err = comparators.gen_func(cmp_operator, key_parts)
+function comparators.gen_tuples_comparator(cmp_operator, key_parts, field_names, space_format)
```

Comment:
It would be great to add unit test for comparators to fix the behaviour. See https://github.com/tarantool/crud/blob/master/test/unit/select_comparators_test.lua.